stl/inc/filesystem

```diff
@@ -275,8 +275,8 @@ namespace filesystem {
 #ifdef __cpp_char8_t
     template <class _Conversion>
     _NODISCARD wstring _Convert_stringoid_to_wide(const basic_string_view<char8_t> _Input, _Conversion) {
-        static_assert(
-            is_same_v<_Conversion, _Normal_conversion>, "invalid value_type, see N4810 D.17 [depr.fs.path.factory]/1");
+        static_assert(is_same_v<_Conversion, _Normal_conversion> || is_same_v<_Conversion, _Utf8_conversion>,

```

Comment:
Should this use `_Is_any_of_v` to be consistent with
https://github.com/microsoft/STL/blob/312047cf3ef2d460ac0e11bf604ca8e3e0e21744/stl/inc/filesystem#L257-L259
?